### PR TITLE
[stable/goldpinger] Deprecate goldpinger chart

### PR DESCRIPTION
**Pull request by okgolove**
Signed-off-by: Mikhail Naletov <okgolove@markeloff.net>

#### Is this a new chart
No

#### What this PR does / why we need it:
Deprecate Goldpinger chart
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/goldpinger/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: goldpinger
-version: 2.0.2
+version: 2.0.4
```
Comment (cpanato) [inline stable/goldpinger/Chart.yaml:7]: please follow this: https://github.com/helm/charts/blob/master/PROCESSES.md#deprecating-a-chart

**Review by cpanato — CHANGES_REQUESTED** [review]

**Comment by okgolove** [comment]
@cpanato added info about deprecation to the chart files.

**Review by cpanato — APPROVED** [review]
thanks!
/lgtm
/approve

**Comment by cpanato** [comment]
@okgolove i think you need to rebase to retrigger CI

**Pull request merged**